pypesto/result/sample.py

```diff
@@ -5,19 +5,113 @@
 import numpy as np
 
 
-class McmcPtResult(dict):
+class SampleResultBase(dict):
+    """
+    Sampling result object base.
+
+    Parameters
+    ----------
+    trace_x: [n_samples, n_par]
+        Parameters.
+        (Caution: indexing is different for McmcPtResult)
+    trace_neglogpost: [n_samples]
+        Negative log posterior values.
+        (Caution: indexing is different for McmcPtResult)
+    trace_neglogprior: [n_samples]
+        Negative log prior values.
+        (Caution: indexing is different for McmcPtResult)
+    burn_in: int
+        The burn in index.
+    time: float
+        The computation time.
+    auto_correlation:
+        The estimated autcorrelation.
+    effective_sample_size:
+        The estimated effective sample size.
+    message: str
+        Textual comment on the profile result.
```

Comment:
```suggestion
        Textual comment on the sampling result.
```

Reply:
:D copied from the original code, which apparently copied it :D